composer.json

```diff
@@ -31,9 +31,9 @@
         "drupal/console": "^1.9",
         "drupal/content_lock": "^2.2",
         "drupal/content_moderation_notifications": "^3.4",
-        "drupal/core-composer-scaffold": "^9.1",
-        "drupal/core-project-message": "^9.1",
-        "drupal/core-recommended": "^9.1",
+        "drupal/core-composer-scaffold": "9.4.14",
```

Comment:
Why not "^9.4"? (Genuine question.)

Reply:
@akf 
This was the command recommended by drupal.org:

```
composer require drupal/core-recommended:9.4.14 \
                 drupal/core-composer-scaffold:9.4.14 \
		drupal/core-project-message:9.4.14 \
		--update-with-all-dependencies
               # I added --ignore-platform-reqs, per ususal
```

The Security Alert page from drupal.org recommended that specific version, which provided that command line.

https://www.drupal.org/sa-core-2023-005